public/javascripts/components/conversation/messageBubble.js

```diff
@@ -0,0 +1,68 @@
+window.ChubboChat.components.messageBubble = Vue.extend({
+  props: ['message', 'index', 'max-index'],
+  template: `
+    <div :class="[classNames.row, focusClass]">
```

Comment:
focusClass for whether or not it wil get focused by conversation page in the $nextTick function